plutus-errors/allErrors.txt

```diff
@@ -0,0 +1,268 @@
+
+
+src/Wallet/Rollup/Render.hs"Could not find referenced PubKeyHash: " <> Text.pack (show pkh)
+
+src/Wallet/Types.hs
+
+GenericAssertion
+WrongVariantError
+
+data ContractError =
+    WalletError WalletAPIError
+    | EmulatorAssertionError AssertionError -- TODO: Why do we need this constructor
+    | OtherError T.Text
+    | ConstraintResolutionError MkTxError
+    | ResumableError MatchingError
+    | CCheckpointError CheckpointError
+
+
+data NotificationError =
+    EndpointNotAvailable ContractInstanceId EndpointDescription
+    | MoreThanOneEndpointAvailable ContractInstanceId EndpointDescription
+    | InstanceDoesNotExist ContractInstanceId
+    | OtherNotificationError ContractError
+    | NotificationJSONDecodeError EndpointDescription Aeson.Value String -- ^ Indicates that the target contract does not have the expected schema
+
+
+src/Language/Plutus/Contract/StateMachine.hs
+data SMContractError s i =
+    InvalidTransition s i
+    | NonZeroValueAllocatedInFinalState
+    | ChooserError Text
+    | SMCContractError ContractError
+
+src/Language/Plutus/Contract/Checkpoint.hs
+data CheckpointError = JSONDecodeError Text
+
+
+src/Language/Plutus/Contract/Effects/RPC.hs
+data RPCCallError =
+    RPCCallError NotificationError
+    | RPCOtherError ContractError
+    | RPCAwaitResponseError ContractError
+
+data RPCRespondError =
+    RPCEndpointError ContractError
+    | RPCNotifyError NotificationError
+
+src/Wallet/Emulator/Error.hs
+data WalletAPIError =
+    InsufficientFunds Text
+    -- ^ There were insufficient funds to perform the desired operation.
+    | PrivateKeyNotFound PubKeyHash
+    -- ^ The private key of this public key hahs is not known to the wallet.
+    | OtherError Text
+    -- ^ Some other error occurred.
+
+src/Language/Plutus/Contract/Trace.hs
+data TraceError e =
+    TraceAssertionError EM.AssertionError
+    | TContractError e
+    | HandleBlockchainEventsMaxIterationsExceeded Wallet MaxIterations
+    | HookError EndpointError
+
+data EndpointError =
+    EndpointNotActive (Maybe Wallet) EndpointDescription
+    | MoreThanOneEndpointActive EndpointDescription
+
+src/Cardano/Metadata/Types.hs
+data MetadataError
+    = SubjectNotFound Subject
+    | PropertyNotFound Subject PropertyKey
+
+
+src/Plutus/SCB/Types.hs
+data SCBError
+    = FileNotFound FilePath
+    | ContractNotFound FilePath
+    | ContractInstanceNotFound ContractInstanceId
+    | SCBContractError ContractError
+    | WalletClientError ClientError
+    | NodeClientError ClientError
+    | MetadataClientError ClientError
+    | MetadataError Metadata.MetadataError
+    | SigningProcessError ClientError
+    | ChainIndexError ClientError
+    | WalletError WalletAPIError
+    | ContractCommandError Int Text
+    | InvalidUUIDError  Text
+    | OtherError Text
+    | EndpointCallError ContractInstanceId EndpointError
+
+src/Plutus/SCB/Webserver/Types.hs ErrorResponse Text
+
+
+Servant ServerError
+Servant ClientError
+
```

Comment:
I'm guessing you didn't mean to commit this either.

Reply:
This is all the errors that I collected. I want to commit it so I don't lose it. Perhaps there is a better place for it.